pkg/scanner/local/scan.go

```diff
@@ -100,6 +101,76 @@ func (s Scanner) Scan(ctx context.Context, targetName, artifactKey string, blobK
 	return s.ScanTarget(ctx, target, options)
 }
 
+func postProcessApplications(apps []ftypes.Application, options types.ScanOptions) []ftypes.Application {
```

Comment:
Would it make sense to add a comment which highlights the purpose of this method

Reply:
Addressed